htdocs/changesshkey.php

```diff
@@ -148,7 +148,7 @@
 #==============================================================================
 # Notify password change
```

Comment:
Could you also update comment?

Reply:
sure!

Reply:
... and I managed to submit another one ... https://github.com/ltb-project/self-service-password/pull/542
not fixing comments ...